cmd/dbseed/main.go

```diff
@@ -441,10 +441,48 @@ func (s *Seeder) SeedData() {
 			"cmmiGroups":      []string{"PATIENT_CARE_MODELS_GROUP", "SEAMLESS_CARE_MODELS_GROUP"},
 			"completeICIP":    "2020-05-13T20:47:50.12Z",
 			"phasedIn":        true,
-			"clearanceStarts": now.AddDate(0, 3, 0),
```

Comment:
was this change needed?

Reply:
This is part of an ugly merge. This is a good catch because it actually should be the `now` variable reused for assertions to avoid false negatives.